extensions/emmet/src/abbreviationActions.ts

```diff
@@ -161,13 +164,26 @@ function doWrapping(individualLines: boolean, args: any) {
 		let extractedResults = helper.extractAbbreviationFromText(inputAbbreviation);
 		if (!extractedResults) {
 			return Promise.resolve(inPreview);
-		} else if (extractedResults.abbreviation !== inputAbbreviation) {
-			// Not clear what should we do in this case. Warn the user? How?
+		} else if (extractedResults.abbreviation !== inputAbbreviation.replace(/\|[a-zA-Z]+/g, '')) {
```

Comment:
Sanity check: is the regex taking out filters? Might be worth adding a comment about it